node/network/availability-recovery/src/tests.rs

```diff
@@ -249,7 +287,6 @@ impl TestState {
 				let _ = tx.send(if with_data {
 					Some(self.available_data.clone())
 				} else {
-					println!("SENDING NONE");
```

Comment:
why not use logging?

Reply:
This seems to be a leftover, I'll fix it.